core/src/main/java/feign/Contract.java

```diff
@@ -312,4 +315,27 @@ protected boolean processAnnotationsOnParameter(MethodMetadata data, Annotation[
       return result;
     }
   }
+
+  class DelegatingContract implements Contract, Param.Expander.Factory {
```

Comment:
Seems a little strange to me. Why can't Contact have an expander factory with accessors? Composition over inheritance.